src/pivot-table/components/PivotTable.tsx

```diff
@@ -82,68 +99,113 @@ export const StickyPivotTable = ({
   const contentCellRowHightCallback = useCallback(() => contentCellHeight, [contentCellHeight]);
 
   return (
-    <ScrollableContainer ref={scrollableContainerRef} rect={tableRect} onScroll={onScrollHandler}>
+    <ScrollableContainer
+      ref={verticalScrollableContainerRef}
+      width={tableRect.width}
+      height={tableRect.height}
+      onScroll={onVerticalScrollHandler}
+      showVerticalScrollbar
+      showHorizontalScrollbar={false}
+      origin="containerGrid"
+    >
       <FullSizeContainer width={totalWidth} height={containerHeight}>
-        <StickyContainer rect={tableRect} leftColumnsWidth={leftGridWidth} rightColumnsWidth={rightGridWidth}>
+        <StickyContainer
+          width={tableRect.width}
+          height={tableRect.height}
+          style={{
+            display: "grid",
+            gridTemplateColumns: leftGridWidth // If leftColumnsWidth is 0, this means no data exist for "headers" or "left"
+              ? `${leftGridWidth}px ${rightGridWidth}px`
+              : `${rightGridWidth}px`,
+          }}
+        >
           {Boolean(leftGridWidth) && (
-            <div style={getLeftGridStyles(leftGridWidth)}>
-              <HeaderGrid
-                columnWidths={leftGridColumnWidths}
-                getHeaderCellsIconsVisibilityStatus={getHeaderCellsIconsVisibilityStatus}
-                rowHight={headerCellHeight}
-                height={topGridHeight}
-                headersData={headersData}
-                translator={translator}
-                changeSortOrder={changeSortOrder}
-                changeActivelySortedHeader={changeActivelySortedHeader}
-              />
+            <ScrollableContainer
+              ref={horizontalScrollableContainerRef}
+              width={leftGridWidth}
+              height={rowsCanFitInTableViewPort ? topGridHeight + dataGridHeight + GRID_BORDER : tableRect.height}
```

Comment:
Seems like this is use in two places. Maybe move it to a const?

Reply:
Yes, and this one will be addressed in next PR (scrollbar overlays on data)